### ISSUE TITLE
[Visual Requirements-Embedded-Create Profile]: TalkBack does not announce anything when page is getting loaded.

### ISSUE THREAD

**Issue by billycamp** [Microsoft/EmbeddedSocial-Android-SDK#11]
"Priority: High

Repro Steps
1. Install ""One Bus Away"" app on Android.  
2. Start ""One Bus Away"" app from Android phone.
3. ""One Bus Away "" Home page will get appear.
4. Navigate to ""Open Navigation Drawer"" button and press ENTER (or double tap) to open.
5. Navigate to ""Sign In"" and press ENTER (or double tap) to open. 
6. Navigate to ""Sign In with google"" and press ENTER (or double tap) to open.
7. Login with valid google account credentials.
8. “Create Profile” page will get appear.
9. Fill all the input fields with valid inputs.
10. Navigate to ""Done"" button and press enter.
11. Verify the speech of TalkBack when page is loading.

Expected
TalkBack should convey the information when page is loading, so that screen reader user can identify the status of the loading page.

Actual
TalkBack does not provide any information about the page spinner when page is getting loaded.

MAS Rule: MAS 1.3.3

Impact
User will get no idea what is happening  on the page if screen reader does not provide any information when page is loading.

Test Environment
OS: Android
Device Model No. : Nexus 5
Android Version : 6.0.1
App: Social Embedded
App Version : v2.2.3 (84)
Tool : Talk Back"
